src/components/ContentItem/ContentItem.stories.js

```diff
@@ -95,6 +95,18 @@ const withIcon = () => ({
     device: {
       default: select('Simulated Mobile Device', [...availableDevices], availableDevices[0]),
     },
+    hasForwardButton: {
+      default: boolean('Has Forward Button?', true),
+    },
+    success: {
+      default: boolean('Has Success forward icon?', true),
```

Comment:
(nit) please update this label to: `'Has Success icon?'`

Reply:
I think more accurate label could be: `Is in Success status?`. It does affect to more stuffs than only the icon, thats why I suggest this one instead. WDYT?

Reply:
ah, good point! then how about `"Has Success  Status?"`